tests/testPrimeGenerator.js

```diff
@@ -0,0 +1,19 @@
+var chai = require('chai');
+var expect = require('chai').expect;
+var numberToGenerate = require('../primeNumber/prime.js');
+
+describe('Generate Primes', function() {
+  beforeEach(function() {
+    console.log('Running Test: ');
```

Comment:
You should be able to kill this. Mocha has different test reporters available if you want more verbose output.